tests/sample_size/scripts/test_multiple_testing.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+from sample_size.sample_size_calculator import SampleSizeCalculator
+
+
+class MultipleTestingTestCase(unittest.TestCase):
+    @patch("sample_size.sample_size_calculator.SampleSizeCalculator._get_single_sample_size", side_effect=[100, 1000])
+    @patch("sample_size.multiple_testing.MultipleTestingMixin._expected_average_power", side_effect=[0.5, 0.8])
+    def test_get_multiple_sample_size(self, mock_expected_average_power, mock_get_single_sample_size):
+        test_lower = 100
+        test_upper = 1000
+        test_metric_type = "boolean"
+        test_probability = 0.05
+        test_mde = 0.02
+        test_metric_metadata = {"probability": test_probability, "mde": test_mde}
+        calculator = SampleSizeCalculator()
+        calculator.register_metrics(
+            [
+                {"metric_type": test_metric_type, "metric_metadata": test_metric_metadata},
+                {"metric_type": test_metric_type, "metric_metadata": test_metric_metadata},
+            ]
```

Comment:
Might be good move almost all of this code to a [`setUp` function](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp) that can be reused across your tests